hubmap/query_app/validation.py

```diff
@@ -343,9 +343,9 @@ def process_evaluation_args(query_params):
     if (
         "limit" not in query_params.keys()
         or not query_params["limit"].isdigit()
-        or int(query_params["limit"]) > 100000
+        or int(query_params["limit"]) > 200000
```

Comment:
I'd set this limit in something like `settings.py` to avoid having to change it multiple times.